community/io/src/main/java/org/neo4j/io/pagecache/IOLimiter.java

```diff
@@ -107,8 +107,27 @@ default void enableLimit()
      * An IOPSLimiter implementation that does not restrict the rate of IO. Use this implementation if you want the
      * flush to go as fast as possible.
      */
-    static IOLimiter unlimited()
+    IOLimiter UNLIMITED = new IOLimiter()
     {
-        return ( previousStamp, recentlyCompletedIOs, flushable ) -> previousStamp;
+        @Override
+        public long maybeLimitIO( long previousStamp, int recentlyCompletedIOs, Flushable flushable )
+        {
+            return previousStamp;
+        }
+
+        @Override
+        public boolean isLimited()
+        {
+            return false;
+        }
+    };
+
+
+    /**
+     * @return {@code true} if IO is currently limited
+     */
+    default boolean isLimited()
+    {
+        return true;
     }
```

Comment:
If we invert this default so it says `false`, like I did in #11804, then we don't have to do many of the changes where lambdas unfurl into anonymous classes, including the static-method `unlimited` into static-field `UNLIMITED`.

Reply:
There was only 2 unfurls, but I can get rid of one by doing that.